apps/omg_watcher/test/omg_watcher/exit_processor/exit_info_test.exs

```diff
@@ -0,0 +1,56 @@
+# Copyright 2019-2020 OmiseGO Pte Ltd
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+defmodule OMG.Watcher.ExitProcessor.ExitInfoTest do
+  @moduledoc """
+  Test of the logic of the exit_info module
+  """
+  use OMG.Watcher.ExitProcessor.Case, async: true
+
+  alias OMG.Watcher.ExitProcessor.ExitInfo
+
+  import OMG.Watcher.ExitProcessor.TestHelper
+
+  @recently_added_keys [:root_chain_txhash]
+  @utxo_pos_1 {0, 0, 0}
+  @exit_1 %{
+    exit_id: 1,
+    amount: 1,
+    currency: random_bytes(20),
+    eth_height: 1,
+    exiting_txbytes: random_bytes(32),
```

Comment:
This is misleading as it contains transaction bytes but suggest ~hash value. Let's make it obviously invalid
```suggestion
    exiting_txbytes: "not important tx bytes",
```